fmpz_mod_mpoly/equal_fmpz.c

```diff
@@ -60,7 +60,7 @@ int fmpz_mod_mpoly_equal_si(
         ulong uc;
 
         if (c == 0)
-            return 0;
+            return 1;
```

Comment:
The commit message and PR description talk about missing `inlines.c`, but this is a change in behavior that is not covered by the description at all. Did it slip in here by accident? Or if it is intentionally in here, then I assume it is a bug fix that should be mentioned ?

Reply:
OK, but is it meant to be in this PR? In that case I would suggest to update the PR title to mention it?

Reply:
yes